### ISSUE TITLE
fmu design matrix: Not only min/max distributions

### ISSUE THREAD

**Issue by bkhegstad** [equinor/fmu-tools#150]
After assisting several assets with uncertainty analysis in general and the design matrix in particular, I see that they often translates high/low to min/max since that is the only option available. The "default" method in Equinor is that high/low at least should be P10/P90 unless clearly stated why. The result is that assets tends to underestimate uncertainties.

Request: Adjust/add functionality to the triangular, pert and uniform distributions (and perhaps the normal one as well) so the input can be set to P90/P10 and not only min/max. An even better solution would be that the user can choose if the high case is P10, P5, P1, P20 or whatever.

**Comment by tralsos** [comment]
Yes, this has been discussed earlier and it will be simpler than going via other software tools to find what the distribution parameters should be if what you have is p10/p90 numbers. To implement this, there is a trade off here between flexibility and keeping the input excel sheet as simple as possible.  There are two different options as I see it:
* Adding new distributions called e.g. uniform_p10_p90 where you specify distribution parameters as p10/P90 instead of min/max. This will probably cover the need in most cases.
* Adding additional distribution parameter columns where you specify which percentiles the "min"/"max" correspond to
The latter is a more flexible solution, but might also be more confusing/prone to errors.
We could e.g. let the uniform distribution have parameters (low, high, percentile_low, percentile_high) where the last two are optional, and do the same for the triangular and pert. We would then have to increase the number of dist_param columns from 4 to 5 (or even 7 for truncated normal distributions?). Alternatively add two optional columns that specify the which percentile the numbers correspond to, but that are only possible to use for the distributions where this is implemented.

**Comment by bkhegstad** [comment]
I think GeoX implemented a _P10P90-version of the beta-distribution after request from us. This is much, much better than having only min/max which tends to underestimate uncertainties by default.   I agree to make it more flexible is a tradeoff between flexibility as easy-to-use.
